build.gradle

```diff
@@ -432,7 +438,7 @@ task stageRelease {
     exec {
       // We need to spawn a new gradle build process in order to upload appropriately
       // More details: http://stackoverflow.com/questions/31614735/gradle-uploadarchives-artificats-namespace-when-depending-on-a-plugin
-      commandLine './gradlew', 'uploadArchives'
+      commandLine './gradlew', 'publishMavenJavaPublicationToMavenRepository'
```

Comment:
`publishMavenJavaPublicationToMavenRepository` comes from gradle plugin itself, but it does not make it less preposterous. It is impossible to remember. Maybe add an `uploadArchives` "alias"?

Reply:
Actually `./gradlew publish` will execute all publication configurations, so that's the one you would want to execute. I could change the line here to that.

I think adding `uploadArchives` would be quite confusing from a maintenance perspective.